File: algorithms/reverse_string/reverse_string.py
def reverseString(arr):
    i = 0
    j = len(arr) - 1
    while i < j:
        tmp = arr[i]
        arr[i] = arr[j]
        arr[j] = tmp
        i += 1
        j -= 1
    return arr

print(reverseString(["h", "e", "l", "l", "o"]))

File: algorithms/reverse_string/reverse_string.js
function reverseString(string) {
  for (let i = 0, j = string.length - 1; i < j; i++, j--) {
    let tmp = string[i];
    string[i] = string[j];
    string[j] = tmp;
  }
  return string;
}

console.log(reverseString(["h", "e", "l", "l", "o"]));
